### ISSUE TITLE
[Project Addition] : Fake and Real Face Detection using DL

### ISSUE THREAD

**Issue by kyra-09** [abhisheks008/DL-Simplified#725]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Fake-Vs-Real-Face Detection</br>
:red_circle: **Aim** : To identify fake and real faces which present a harder challenge to classify them correctly even for the human eye.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/hamzaboulahia/hardfakevsrealfaces </br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Aaradhya Singh
- GitHub Profile Link : https://github.com/kyra-09
- Email ID : aaradhyasinggaur@gmail.com
- Participant ID (if applicable):
- Approach for this Project :
   1. Data Preparation: Collect and preprocess a balanced dataset of real and fake face images, including normalization, resizing, and augmentation.
  2.  Base Model Selection:  EfficientNetB0,VGG16 , Xception , InceptionV3 like 5 different models  excluding its top layers, to leverage its learned features.
  3.  Model Construction: Add custom layers on top of the base model for binary classification, compiling with appropriate loss and metrics.
  4.  Initial Training: Train the model with the base layers frozen to only update the new layers.
  5.  Fine-Tuning: Unfreeze some or all of the base model layers and continue training with a lower learning rate to fine-tune the entire network.
  6.) EDA analysis.
  7.) Comaprioson using performance matrices such as accuracy scores , confusion matrix etc.
- What is your participant role? (Mention the Open Source program)  - GSSOC-2024 Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Already assigned to an issue.

**Comment by kyra-09** [comment]
@abhisheks008 kindly assign this issue to me . I have worked on it.

**Comment by abhisheks008** [comment]
Just assigned you the previous issue. You can work on this only after completing the previous issue.

**Comment by abhisheks008** [comment]
Hi @kyra-09 updated the issue name. Please follow the same as the project folder name.

Assigned @kyra-09
